operator/api/v1beta1/lokistack_types.go

```diff
@@ -725,6 +768,35 @@ type LokiStackComponentStatus struct {
 	Ruler PodStatusMap `json:"ruler,omitempty"`
 }
 
+// StorageSchemaStatus defines the observed state of a single
+// change to the Loki Storage Schema.
+type StorageSchemaStatus struct {
```

Comment:
Although they are the same, it think it's fine to have the two objects. Putting a `Spec` labeled object in `Status` or vice versa seems confusing. It might be odd to have a non-spec labeled object in the `Spec`.